jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamDebugSupport.kt

```diff
@@ -26,17 +27,29 @@ interface SamDebugSupport {
     fun createDebugProcessAsync(
         environment: ExecutionEnvironment,
         state: SamRunningState,
+        debugHost: String,
         debugPorts: List<Int>
-    ): Promise<XDebugProcessStarter?> = resolvedPromise(createDebugProcess(environment, state, debugPorts))
+    ): Promise<XDebugProcessStarter?> = resolvedPromise(createDebugProcess(environment, state, debugHost, debugPorts))
 
     fun createDebugProcess(
         environment: ExecutionEnvironment,
         state: SamRunningState,
+        debugHost: String,
         debugPorts: List<Int>
     ): XDebugProcessStarter?
 
     fun isSupported(): Boolean = true
 
+    fun getDebugHost(): String {
+
+        val localDebugHost = LocalDebugSettings.getInstance().localDebugHost
```

Comment:
Could do this in a single statement with the Elvis operator.

Reply:
Noted, I can convert this to make use of the Elvis operator here, I chose to KISS, however will update the PR.